Fixtures/BinaryTargets/Framework.xcframework/Info.plist

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>AvailableLibraries</key>
+	<array>
+		<dict>
+			<key>LibraryIdentifier</key>
+			<string>macos-x86_64</string>
+			<key>LibraryPath</key>
+			<string>Framework.framework</string>
```

Comment:
Can you call it something else? `Framework.framework` is a bit confusing 😁

Reply:
I called it that way to clearly differentiate it from the static and dynamic library binaries. Do you have a suggestion?

Reply:
`MyFwk`?